app/main.py

```diff
@@ -1 +1,30 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+
+    mv, source_file, destination = command.split()
```

Comment:
The `command.split()` assumes that the command string will always have exactly three parts. Consider adding error handling to manage cases where the command string does not meet this expectation.